test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -628,6 +629,7 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 							fmt.Sprintf("stalld is not running on %q when it should", node.Name))
 
 						By(fmt.Sprintf("Checking TuneD parameters on %q", node.Name))
+						kernelParameters = append(kernelParameters, utilstuned.AddPstateParameter(context.TODO(), node))
```

Comment:
Hi @MarSik, so after the recent tuned merge, the var `intel_pstate= {f:intel_recommended_pstate}` is expanding to either `active` or `disable`.  It is not causing any issue for render test cases, but for `e2e-gcp-pao-workloadhints` a real cluster is deployed so the variable is expanding. The problem is, i don't think we can anticipate where these tests are running, therefore i added a check with `func AddPstateParameter()` to check the pmu_name and add the corresponding pstate mode in the kernel Parameters. But it seems the file doesn't exist on the node and the test fails with:

```
[FAILED] failed to execute command ["cat" "/sys/devices/cpu/caps/pmu_name"] on node: "ci-op-lrm0rmvm-36360-nncwg-worker-a-5nnq6"
  Unexpected error:
      <*fmt.wrapError | 0xc00059e160>: 
      failed to run command [cat /sys/devices/cpu/caps/pmu_name]: output "cat: /sys/devices/cpu/caps/pmu_name: No such file or directory\r\n"; error ""; command terminated with exit code 1
      {
          msg: "failed to run command [cat /sys/devices/cpu/caps/pmu_name]: output \"cat: /sys/devices/cpu/caps/pmu_name: No such file or directory\\r\\n\"; error \"\"; command terminated with exit code 1",
          err: <exec.CodeExitError>{
              Err: <*errors.errorString | 0xc0005120a0>{
                  s: "command terminated with exit code 1",
              },
              Code: 1,
          },
      }
      ```
      
      @mrniranjan Do these tests get executed on baremetal nodes or vms?

Reply:
They get executed on VM's and also on BM"s (but manually)

Reply:
Yeah, the test should skip gracefully on VMs (or figure out it is a VM and test for the proper value = disable?)

Reply:
I think, i found it how to verify if it is a [VM](https://github.com/openshift/cluster-node-tuning-operator/blob/dd2698c43517191d17dc5c4ac842e6d4dc3aab32/test/e2e/performanceprofile/functests/utils/infrastructure/vm.go#L13)

Reply:
@sabbir-47 That code is broken, since the systemd-detect-vm is executed from a container, it will always return podman/container and the test will skip (even on BM)

Reply:
Well, is there any better way to verify then? I can hardcode the pstate to disable  in the kernel parameters and the test will always pass if the tests are executed on VM. It will fail if the tests are run on BM(IceLake+ servers), where it expects to set the pstate to active.

Reply:
we have to fix that code, for now just check the number of cpus or execute lscpu and check  Bios Vendor id , if it's QEMU it's a VM

Reply:
All tests passed with recent changes.